fix: clarify partition labels (#3855)
scaleway/docs-content: bare-metal/elastic-metal/how-to/configure-disk-partitions.mdx

- Disks:
    - Each disk is specified with its device path (e.g., `/dev/nvme0n1` or `/dev/nvme1n1`).

>     - Partitions are defined with labels. Default value is `unknown_partition_label`, and possible values are: `uefi`, `legacy`, `root`, `boot`, `swap`, `data`, `home`, `raid`. Refer to [API documentation](https://www.scaleway.com/en/developers/api/elastic-metal/#path-servers-install-an-elastic-metal-server) if you have a doubt.

Reviewer: ```suggestion
    - Partitions are defined with labels. The default value is `unknown_partition_label`, and possible values are: `uefi`, `legacy`, `root`, `boot`, `swap`, `data`, `home`, `raid`. Refer to the [API documentation](https://www.scaleway.com/en/developers/api/elastic-metal/#path-servers-install-an-elastic-metal-server) for full details.
```
